pkg/app/piped/cmd/piped/piped.go

```diff
@@ -99,6 +104,7 @@ func NewCommand() *cobra.Command {
 	cmd.Flags().StringVar(&p.toolsDir, "tools-dir", p.toolsDir, "The path to directory where to install needed tools such as kubectl, helm, kustomize.")
 	cmd.Flags().BoolVar(&p.useFakeAPIClient, "use-fake-api-client", p.useFakeAPIClient, "Whether the fake api client should be used instead of the real one or not.")
 	cmd.Flags().BoolVar(&p.enableDefaultKubernetesCloudProvider, "enable-default-kubernetes-cloud-provider", p.enableDefaultKubernetesCloudProvider, "Whether the default kubernetes provider is enabled or not.")
+	cmd.Flags().BoolVar(&p.addLoginUserToPasswd, "add-login-user-to-passwd", p.addLoginUserToPasswd, "Whether to add login user to /etc/passwd. This is typically for applications running as a random user ID.")
```

Comment:
nit: Actually it adds to "$HOME/psswd"

Reply:
For sure.

Reply:
In that case, I think it would be better to rename this to something that expresses the purpose of this flag instead of showing what we do 🤔 Maybe, `registerApplicationUser` if I guess right 🤔

Reply:
Probably @nghialv is talking about just the description, you know?

Reply:
Yes, I was talking about the description.

Reply:
Yeh, I get that point 👍 But what I mean is instead of using description as "This is typically for applications running as a random user ID." maybe rename this flag name so that user could understand what is this flag stands for is a better option 🤔

Reply:
😢 Approved the PR before reading @khanhtc1202 's latest message. I get your point. But actually, I don't have a strong opinion on that. That is an edge case and that name is fine to me.

Reply:
I know what @khanhtc1202 means, but me neither 😢

Reply:
I see 🙏 Since we don't expect users to use this flag, let's keep the name as is. Sorry for the interruption 😄

Reply:
But you have a point. I tried to look for a suitable name but couldn't. `addLoginuserToPasswd` represents what it does well so I compromised ;).

Reply:
>Sorry for the interruption

No, nice point

Reply:
> But you have a point. I tried to look for a suitable name but couldn't to be honest. it's better to tell the behavior with only its name. but it's for the edge case and `addLoginuserToPasswd` represents what it does well so I compromised ;).

Thanks for the clarification, I got stuck too 😂